src/ESP01Firmware/ESP01Firmware.ino

```diff
@@ -35,4 +35,12 @@ void loop() {
     }
     client.stop();
   }
-}
\ No newline at end of file
+}
+
+// Returns a suffix based on the NIC specific part of the MAC address.
+String getSuffix() {
+  String mac = WiFi.macAddress().substring(9, 17); // remove vendor ID
+  mac.remove(2, 1);                                // remove first colon
+  mac.remove(4, 1);                                // remove second colon
+  return "_" + mac;
+}
```

Comment:
Stylewise, I'd prefer

- A doxygen comment before the function
- snake_case function name
- Comments on separate lines 

I think we can use `.replace(":", "")` instead of the two `.remove` calls, but I haven't tested it.

```suggestion
/**
 * @brief Return a suffix based on the NIC specific part of the MAC address
 *
 * @return The final three octets of the MAC address, prefixed with underscore
 */
String get_suffix()
{
   // Remove vendor ID
  String mac = WiFi.macAddress().substring(9, 17);
  // Remove colons
  mac.replace(":", "")
  return "_" + mac;
}
```